pkg/util/scrubber/writer.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package scrubber
+
+import (
+	"bufio"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+// BUG(dustin) Writer applies scrubbing to each "chunk" of data independently. If
```

Comment:
Should this be addressed before this merges?

Reply:
No, in fact I don't know if it *can* be addressed.  It's a longstanding bug, anyway.

Reply:
Go it - was just asking since it has your name in it.  I wasn't sure it was something we should check in.